### ISSUE TITLE
error: ../../../.pub-cache/hosted/pub.dev/dio-5.4.3/lib/src/form_data.dart:175:54: Error: The argument type 'List<int>' can't be assigned to the parameter type 'Uint8List'.

### ISSUE THREAD

**Issue by shashankmathur8** [cfug/dio#2184]
### Package

dio

### Version

5.4.3

### Operating-System

iOS

### Adapter

Default Dio

### Output of `flutter doctor -v`

```shell
! Doctor found issues in 1 category.
shashank@Shashanks-MacBook-Air bluebits-mobile % flutter doctor -v
[✓] Flutter (Channel stable, 3.7.10, on macOS 13.6 22G120 darwin-arm64, locale en-IN)
    • Flutter version 3.7.10 on channel stable at /Users/shashank/Documents/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4b12645012 (1 year ago), 2023-04-03 17:46:48 -0700
    • Engine revision ec975089ac
    • Dart version 2.19.6
    • DevTools version 2.20.1

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/shashank/Library/Android/sdk
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.13.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] Connected device (3 available)
    • iPad (10th generation) (mobile) • 58251097-0265-42AE-A0E5-9A5ECE1091B7 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)
    • macOS (desktop)                 • macos                                • darwin-arm64   • macOS 13.6 22G120 darwin-arm64
    • Chrome (web)                    • chrome                               • web-javascript • Google Chrome 123.0.6312.123

[✓] HTTP Host Availability
    • All required HTTP hosts are available
```

### Dart Version

2.19.6

### Steps to Reproduce

Build app 
Get this error

### Expected Result

No Error and build should complete successfully

### Actual Result

error: ../../../.pub-cache/hosted/pub.dev/dio-5.4.3/lib/src/form_data.dart:175:54: Error: The argument type 'List<int>' can't be assigned to the parameter type 'Uint8List'.

**Comment by phil10xs** [comment]
Waiting for this checks to pass. This just stop my project from running

**Comment by AlexV525** [comment]
This is fixed in v5.4.3+1.
